jade/jade.py

```diff
@@ -0,0 +1,20 @@
+"""
+Console access to JADE
+
+Usage:
+    jade [-h | --help]
+    jade server
+
+Available commands:
+    server      Start the web server.
+
+Options:
+    --help      Show this usage message.
+"""
+import docopt_subcommands as dsc
```

Comment:
Holy moly

Reply:
Yeah... this is not something I'm terribly proud of relying on, unless we're going to make some upstream changes.  But it was expedient, and has the nice property of collapsing subcommand help until it's specifically asked for.